install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -200,5 +195,11 @@ DELETE FROM `PREFIX_configuration` WHERE `name` IN (
   'SHOP_LOGO_MOBILE_WIDTH');
 
 ALTER TABLE `PREFIX_tab` ADD `icon` varchar(32) DEFAULT '';
+ALTER TABLE `PREFIX_profile_lang` ADD UNIQUE KEY (`name`);
```

Comment:
yes, to be moved in 1.7.1.0, the new autoupgrade is not allowed to run in 1.7.0.0 anyway